src/parser.py

```diff
@@ -0,0 +1,195 @@
+import xml.etree.ElementTree as ET					# xml.etree.ElementTree module implements an API for parsing and creating XML data.
+from xml.etree.ElementTree import Element
+
+from typing import TypedDict
+
+import numpy as np
+import numpy.typing as npt
+
+class Node_t(TypedDict):
+	posX:			int
+	posY:			int
+	node_id:	int
+	label:		str
+
+class Edge_t(TypedDict):
+	source_id:	int
+	target_id:	int
+	upText:			str
+
+class Positions_t(TypedDict):
+	label:	str
+	x:			int
+	y:			int
+
+class Parser:
+	# XML tree root.
+	root: Element
```

Comment:
Dichiarare una variabile in questa maniera equivale a renderla statica (associata alla classe `Parser`, non al singolo oggetto). Non penso che questo sia l'obiettivo. Basta rimuovere questa riga.

Reply:
@TendTo la mia intenzione era quello di rendere esplicito il fatto che nella classe è presente la proprietà root, non quello di creare una proprietà statica; in pratica volevo dichiarare la proprietà prima di andarla ad utilizzare, c'è un modo per fare questa cosa (a meno di eliminare la riga ovviamente)?